schemas/oathkeeper.config.schema.json

```diff
@@ -375,7 +375,7 @@
         },
```

Comment:
Changes in this file must actually come from ORY Oathkeeper, [which you have done](https://github.com/ory/oathkeeper/pull/331/files#diff-36449d2bff0518bdb98b6dda1dbce16d). However, changes here appear to have more lines changed (several descriptions) than the file at ORY Oathkeeper. Could you maybe update the PR there and revert the changes here? I will then merge the ORY Oathkeeper PR first to ensure that the file is up to date!

Reply:
About to verify the schema file in Oathkeeper PR to be sure